transform/Downscale_local_mean_XGZ/test.py

```diff
@@ -0,0 +1,42 @@
+import matplotlib.pyplot as plt
```

Comment:
Always good to add a header block. Block would include authorship, license, copyright, dates and description of what the driver does.